README.md

```diff
@@ -3,35 +3,70 @@
 ## Prerequisite
 
 ### bpftool & XDP-tools (submodules)
-PV 3.0 mainly uses XDP-tools lib and bpftool is used for compiling XDP-tools lib.
+PV 3.0 mainly uses libxdp in XDP-tools and bpftool is used to set configuration of XDP-tools.
 
 The script, `build.sh`, includes submodule update so that you can do the next step.
 
```

Comment:
Link library as a link.
But, As `bpftool` is built automatically, I think we don't need to mention that bpftool is needed.

```suggestion
PV 3.0 mainly uses libxdp in [XDP-tools][] and `bpftool` is used to set configuration of XDP-tools.

The script, `build.sh`, includes submodule update so that you can do the next step.

[XDP-tools]: https://github.com/xdp-project/xdp-tools
```